### PR TITLE
Prl 6251

### PR DESCRIPTION
**Pull request by Scralfie**
### Change description

Enter the ticket description

### JIRA link

[PRL-6251](https://tools.hmcts.net/jira/browse/PRL-6251)

**Before merging a pull request make sure that:**

- [x] Commits are meaningful and simple
- [x] All commits are squashed into a single commit
- [x] README and other documentation has been updated / added (if needed)

### CODE REVIEW

File: e2e/pages/manageCases/createCase/C100/childrenAndRespondents/childrenAndRespondents1Page.ts [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.GovukFormLabel}:text-is("${ChildrenAndRespondents1Content.formLabelNo}")`,
+        1,
+      ),
+    ]);
+    if (accessibilityTest) {
+      // await AccessibilityTestHelper.run(page);
```
Comment (JayAshton) [inline e2e/pages/manageCases/createCase/C100/childrenAndRespondents/childrenAndRespondents1Page.ts:99]: Add the ticket number as a comment for why these a11y checks are commented please

File: e2e/pages/manageCases/createCase/C100/childrenAndRespondents/childrenAndRespondents1Page.ts [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+        1,
+      ),
+      Helpers.checkGroup(
+        page,
+        2,
+        ChildrenAndRespondents1Content,
+        "formLabel",
+        Selectors.GovukFormLabel,
+      ),
+      Helpers.checkVisibleAndPresent(
```
Comment (JakeCG) [inline e2e/pages/manageCases/createCase/C100/childrenAndRespondents/childrenAndRespondents1Page.ts:67]: Can you make line 67 - 94 a checkgroup instead?

File: e2e/pages/manageCases/createCase/C100/childrenAndRespondents/childrenAndRespondents1Page.ts [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+        1,
+      ),
+      Helpers.checkVisibleAndPresent(
+        page,
+        `${Selectors.GovukErrorMessage}:text-is("${ChildrenAndRespondents1Content.errorMessageDoesChildLiveWithPerson}")`,
+        1,
+      ),
+    ]);
+    await page.waitForTimeout(1000);
+    await page.selectOption(
```
Comment (JakeCG) [inline e2e/pages/manageCases/createCase/C100/childrenAndRespondents/childrenAndRespondents1Page.ts:138]: This should wait for the option to appear, so the waitforTimeout isn't needed

File: e2e/pages/manageCases/createCase/C100/childrenAndRespondents/childrenAndRespondents1Page.ts [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+    } else {
+      await page.selectOption(
+        `${UniqueSelectors.respondentsRelationshipDropdown}`,
+        ChildrenAndRespondents1Content.selectionForOtherRelationship,
+      );
+      await page.fill(
+        `${UniqueSelectors.giveDetailsForOtherRelationshipInput}`,
+        ChildrenAndRespondents1Content.loremIpsum,
+      );
+      await this.giveDetailsOtherRelationshipStaticText(page);
```
Comment (JakeCG) [inline e2e/pages/manageCases/createCase/C100/childrenAndRespondents/childrenAndRespondents1Page.ts:192]: Don't think this needs a separate function as it calls one command

File: e2e/pages/manageCases/createCase/C100/childrenAndRespondents/childrenAndRespondentsSubmitPage.ts [hunk 5]
```diff
@@ -0,0 +1,116 @@
...
+    );
+    await this.continue(page);
+  }
+
+  private static async checkPageContent(
+    page: Page,
+    accessibilityTest: boolean,
+    yesNoChildrenAndRespondents: boolean,
+  ): Promise<void> {
+    await this.checkPageLoads(
```
Comment (JakeCG) [inline e2e/pages/manageCases/createCase/C100/childrenAndRespondents/childrenAndRespondentsSubmitPage.ts:29]: Can you make both of these a Promise.all([]) so the checks run simultaneously?

File: e2e/tests/manageCases/createCase/C100/c100ChildrenAndRespondents.spec.ts [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+      yesNoDontKnowC100ChildDetailsRevisedAdditionalQuestions: "yes",
+      yesNoChildrenAndRespondents: true,
+      subJourney: true,
+    });
+  });
+  test(`Complete the C100 Create case Children and respondents as a solicitor with the following options:
+  Not Accessibility testing,
+  Not Error message testing,
+  Saying no to all options,
+  Setting the applicant Gender to female. @crossbrowserManageCases`, async ({
```
Comment (JakeCG) [inline e2e/tests/manageCases/createCase/C100/c100ChildrenAndRespondents.spec.ts:34]: Remove the @crossbrowserManageCases tag

File: e2e/tests/manageCases/createCase/C100/c100ChildrenAndRespondents.spec.ts [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+      subJourney: true,
+    });
+  });
+});
+
+test(`Complete the C100 Create case Children and respondents as a solicitor with the following options:
+  Accessibility testing,
+  Not Error message testing,
+  Saying Yes to all options,
+  Setting the applicant Gender to female. @crossbrowserManageCases`, async ({
```
Comment (JakeCG) [inline e2e/tests/manageCases/createCase/C100/c100ChildrenAndRespondents.spec.ts:58]: Does this not need the @accessibilityManageCases tag?

**Review by JakeCG — CHANGES_REQUESTED** [review]

**Review by OLT2000 — APPROVED** [review]
lgtm

**Pull request merged**